code/index.html

```diff
@@ -4,19 +4,75 @@
     <meta charset="utf-8" />
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Urbanist:wght@500&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Oswald&display=swap" rel="stylesheet">
     <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
   </head>
   <body>
-    <header>
+    <header class="header">
       <!-- Some kind of logo and navbar -->
+      <img class="logo" src="./logo.PNG" alt="The View Report"/>
+      <nav class="navigation">
+        <a href="#">Search</a>
+        <a href="#">Menu</a>
+        <a href="#">Login</a>
+        <a href="#">Subscribe</a>
+        <div class="burger">
+          <span class="line"></span>
+          <span class="line"></span>
+          <span class="line"></span>
+        </div>
+      </nav>
     </header>
 
-    <section class="big-news">
```

Comment:
I couldn't find this class in the css file. You may consider removing this one as well. :)